content/en/docs/instrumentation/cpp/getting-started.md

```diff
@@ -1,185 +1,476 @@
 ---
 title: Getting Started
+description: Get telemetry for your app in less than 5 minutes!
+cSpell:ignore: oatpp rolldice
 weight: 10
-# prettier-ignore
-cSpell:ignore: Bazel DBUILD devel DWITH helloworld libcurl openssl tracestate xcode
 ---
 
-Welcome to the OpenTelemetry C++ getting started guide! This guide will walk you
-through the basic steps in installing, instrumenting with, configuring, and
-exporting data from OpenTelemetry.
+This page will show you how to get started with OpenTelemetry in C++.
 
-You can use [CMake](https://cmake.org/) or [Bazel](https://bazel.build/) for
-building OpenTelemetry C++. The following getting started guide will make use of
-CMake and only provide you the most essential steps to have a working example
-application (a HTTP server & HTTP client). For more details read
-[these instructions](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/INSTALL.md).
+You will learn how to instrument a simple C++ application automatically,
+such that [traces](/docs/concepts/signals/traces/), [metrics](/docs/concepts/signals/metrics/) and [logs](/docs/concepts/signals/logs/) are emitted to the terminal.
 
-## Prerequisites
+## Prerequisites 
 
-You can build OpenTelemetry C++ on Windows, macOS or Linux. First you need to
-install some dependencies:
+Ensure that you have the following installed locally:
 
-{{< tabpane text=true >}} {{% tab "Linux (apt)" %}}
+- Git
+- C++ compiler supporting C++ version >= 14
+- Make 
+- CMake version >= 3.1
+
+## Example Application 
+
+The following example uses a basic [Oat++](https://oatpp.io/) application. If you are 
+not using Oat++, that's OK - you can use OpenTelemetry C++ with any other web framework as well.
+
+## Setup 
+- Create a folder named `Opentelemetry-Starter-Project`.
+- move into the newly created folder. This will serve as your working directory.
+- Directory Structure: 
+   ```plaintext 
+   Opentelemetry-Starter-Project
+   │
+   ├── oatpp
+   ├── opentelemetry-cpp
+   └── roll-dice
+
+   ```
```

Comment:
Yes, when they do a git clone oatpp and the opentelemetry-cpp folder will be created for them. The only folders they need to create are  Opentelemetry-Starter-Project and roll-dice. Should I make this instruction clear?

Reply:
Yes, this needs to be clearer, otherwise this is confusing for them. Maybe at this point having that directory structure is not required. Just say that they should have `OpenTelemetry-Starter-Project/roll-dice` by now.